nodes/viz/texture_viewer.py

```diff
@@ -236,7 +257,21 @@ def draw_buttons_ext(self, context, layout):
         layout.separator()
         layout.prop(self, "bitmap_format", text='format')
         layout.separator()
-
+        row = layout.row()
+        row.prop(self, 'color_mode_save', expand=True)
+        layout.separator()
+        if img_format == 'PNG':
+            row = layout.row()
+            row.prop(self, 'compression_level', text='set compression')
+            layout.separator()
+        if img_format == 'JPEG':
```

Comment:
Jpeg and jpeg2000 share same quality settings but if i did so:
 if img_format == 'JPEG' or 'JPEG2000': the condition will ever respected and quality button will show for every bitmap format. How we compact them?